src/components/InputForm/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+
+import classes from './style.module.css';
+
+class InputForm extends Component {
+  state = {
+    content: ''
+  }
+
+  changeHandler = event => {
+    this.setState({content: event.target.value});
+  }
+
+  render() {
+    const { content } = this.state;
+    return (
+      <div>
```

Comment:
You can probably scrap the `<div>` since everything is wrapped by `<form>` already.

Reply:
Definitely.